package.json

```diff
@@ -59,48 +60,56 @@
       "js",
       "jsx",
       "json"
-    ]
+    ],
+    "globals": {
+      "ts-jest": {
+        "tsConfigFile": "tsconfig.base.json"
+      }
+    }
   },
   "dependencies": {
-    "babel-plugin-istanbul": "^4.1.6",
-    "babel-preset-jest": "^23.0.0",
+    "closest-file-data": "^0.1.4",
     "cpx": "^1.5.0",
     "fs-extra": "6.0.1",
-    "jest-config": "^23.0.0",
+    "jest-config": "^23.4.1",
     "lodash": "^4.17.10",
     "pkg-dir": "^3.0.0",
     "yargs": "^12.0.1"
   },
   "peerDependencies": {
-    "babel-core": "^6.26.3 || ^7.0.0-0",
-    "jest": "^23.0.0 || ^24.0.0",
+    "babel-core": "^7.0.0-0",
```

Comment:
It's in beta, not alpha. And they are not forced. I've tested some minimal repo not using 7 by just `yarn link ts-jest`, it was working.

Anyway, the big issue is the 😡 move of babel from `babel-xxx` to `@babel/xxx`, big mess in many repo depending on it. The `babel-core ^7.0.0` ensure that if the user has some packages depending on `babel-core` and some others on `@babel/core` it'll use the `@babel/core` one.

Reply:
Isn't this going to end up with a lot of unmet peerDependency warnings though? I know React Native's switched over to babel7 already, but I'm very hesitant about forcing it here.

Reply:
@GeeWee I'm not an expert in npm and deps. If you know a better way to handle this, help more than welcome!

The facts:
- if **anything** has `babel-core` or `babel-preset-env` (old versions) as a dep, even tho you'd try to `yarn add @babel/core` it will bail until you `yarn add babel-jest` (latest).
- `babel` cannot be a dependency, as in the special case where users will set `skipBabel: true`, but mostly so that users can install their own version, same as typescript, jest, ... or should we add them as deps, but locking their versions?

Reply:
No this is a hard issue. It depends whether @kulshekhar is okay breaking backwards compatibillity  (because it *definitely* will, for a lot of people)